src/styles/components/_card.scss

```
@@ -0,0 +1,57 @@
+.catalog {
+  width: 200px;
+  box-sizing: border-box;
+}
+
+.catalog-card {
+  padding: 32px 16px;
+
+  &_image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+  &_title {
+    font-size: 12px;
+    margin-bottom: 4px;
+    font-weight: 500;
+    line-height: 1.5;
+  }
+  &_subtitle {
+    font-size: 10px;
+    color: $secondaryColor;
+    margin-bottom: 16px;
+  }
+}
+
+.catalog-button {
```

```suggestion
.card {
  &__button {
```
As this button is related to card it should an element, not a separate block